Added views and GET/POST methods to add delivery employee to the data…

craigd2387 opened the pull request:
…base

craigd2387 committed:
Added views and GET/POST methods to add delivery employee to the database

shaunganley commented on app.ts:
```diff
@@ -25,6 +28,14 @@ app.use(express.json())
+
+declare module "express-session" {
+    interface SessionData {
+        employee: Employee;
```
you're using a model with an employee ID here which you don't need. should you create a new request model?

shaunganley commented on service/employeeService.ts:
```diff
@@ -13,4 +13,33 @@ module.exports.getemployees = async function (): Promise<Employee[]> {
+
+module.exports.createEmployee = async function (employee:Employee): Promise<number> {
+
+    //const error: string = productValidator.validateProduct(product)
+
+    //if(error){
+      //  throw new Error(error)
+    //}
```
uncomment to add validation

craigd2387 committed:
Merge branch 'main' into 001-create-new-delivery-employee

craigd2387 committed:
001-create-new-delivery-employee: created createEmployee model which
doesn't take ID and created an employeeCreatorValidator with validation error messages
